src/torrent_clients/qbittorrent.py

```
@@ -1466,13 +1381,11 @@ async def _process_torrent_matches(
                             lambda qbt_client=qbt_client, torrent_hash=torrent.hash: asyncio.to_thread(qbt_client.torrents_trackers, torrent_hash=torrent_hash),
                             f"Get trackers for torrent {torrent.name}",
                         )
-                except (TimeoutError, qbittorrentapi.APIError):
-                    if meta.debug:
-                        logger.debug(f"[yellow]Failed to get trackers for torrent {torrent.name} after retries")
+                except TimeoutError, qbittorrentapi.APIError:
+                    logger.debug(f"[yellow]Failed to get trackers for torrent {torrent.name} after retries")
                     continue
                 except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)